intranet/apps/eighth/serializers.py

```diff
@@ -8,7 +8,7 @@
 
 from django.contrib.auth import get_user_model
 from django.db import transaction
-from django.db.models import Count
+from django.db.models import Count, Q
```

Comment:
Why import of `Q`?

Reply:
I needed it in some earlier versions of this code and forgot to remove it. Now removed since it broke the build.